R/repo.R

```diff
@@ -100,12 +100,12 @@ repo_meta_gh_like <- function(link, branch = NULL) {
 }
 
 # adapted from usethis:::github_link()
-parse_github_like_url <- function(link) {
+parse_github_like_url <- function(link, allow_subgroups = FALSE) {
```

Comment:
Can you please remove `allow_subgroups` since you don't use it anywhere?  You'll also need to update the comment above, since this regexp is no longer adapted from `usethis:::github_link()`

Reply:
You overlooked something, of course I was using `allow_subgroups`. I changed the call to use named args, so it hopefully won't get overlooked in the future:

https://github.com/r-lib/pkgdown/blob/a73161aa7bc2562100e512009938455291722c51/R/repo.R#L92-L95

I've also removed the obsolete comment about `usethis:::github_link()`.

Reply:
Why does it need to be an argument? Why can't `parse_github_like_url()` determine whether or not the should allow subgroups?

Reply:
> Why can't `parse_github_like_url()` determine whether or not the should allow subgroups?

It can now and the argument is gone.

I don't fully recall why I chose to make it an argument initially, probably because I thought it would be best to keep it as flexible as possible. But it doesn't really matter, I guess. `parse_github_like_url()` is called exactly 1 time in the whole code anyways.